week2/Week2/src/week2/PrintNumbers.java

```diff
@@ -0,0 +1,24 @@
+
+package week2;
+
+
+/**
+ *
+ * @author sara
+ */
+public class PrintNumbers {
+
+    public static void printNum(){
```

Comment:
Use descriptive name of method to explain the purpose from it:
 ex: printDivisibleByThree()